src/module.ios.ts

```diff
@@ -31,9 +35,12 @@ async function check(permission: Permission): Promise<PermissionStatus> {
     : RESULTS.UNAVAILABLE;
 }
 
-async function request(permission: Permission): Promise<PermissionStatus> {
+async function request(
+  permission: Permission,
+  options?: RequestOptions,
```

Comment:
You have to default to empty object in case of undefined `options`
`options: RequestOptions = {},`

Reply:
See below for the explanation of the crash